### PR TITLE
CalDAV support

### PR DESCRIPTION
**Pull request by olegtsvetkov**
Hi. 

I've implemented basic CalDav support with 3 actions:
1. Fetch events from calendar
2. Create event
3. Delete event

And 2 minor changes:
- Fixed CardDav node path in package.json
- Added icon for CardDav node

### CODE REVIEW

File: nodes/CalDavNode/CalDavNode.node.ts [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+} from 'n8n-workflow';
+import {calendar, event} from "./actions";
+import {loadOptions} from "./methods";
+import {CalDav} from "./actions/Interface";
+
+export class CalDavNode implements INodeType {
+	description: INodeTypeDescription = {
+		displayName: 'CalDAV Node',
+		name: 'calDavNode',
+		icon: 'file:CalDavNode.svg',
```
Comment (TobiasMende) [inline nodes/CalDavNode/CalDavNode.node.ts:16]: I noticed, that the file starts with a lowercase `c` – not sure if that is important, but I think it makes sense to have it consistent.

File: nodes/CardDavNode/CardDavNode.node.ts [hunk 2]
```diff
@@ -13,6 +13,7 @@ export class CardDavNode implements INodeType {
 	description: INodeTypeDescription = {
 		displayName: 'CardDAV Node',
 		name: 'cardDavNode',
+		icon: 'file:CardDavNode.svg',
```
Comment (TobiasMende) [inline nodes/CardDavNode/CardDavNode.node.ts:16]: Should the capitalization here match the capitalization of the file name?

**Review by TobiasMende — COMMENTED** [review]
Thanks so much for your work on this node @olegtsvetkov!

Overall, from a high level, it looks good and like a great first step toward a CalDAV node. 👏 

Left some small comments.

File: nodes/CalDavNode/CalDavNode.node.ts (hunk 1 above)
Comment (olegtsvetkov) [reply nodes/CalDavNode/CalDavNode.node.ts:16]: File on my Mac starts with capital `C`. Committed with `git config core.ignorecase false`. Now config matches filename.

File: nodes/CardDavNode/CardDavNode.node.ts (hunk 2 above)
Comment (olegtsvetkov) [reply nodes/CardDavNode/CardDavNode.node.ts:16]: Fixed

**Pull request merged**